mdbenchmark/generate.py

```diff
@@ -103,10 +115,40 @@ def generate(name, gpu, module, host, min_nodes, max_nodes, time, list_hosts):
             'If you use the {} option make sure you use the GPU compatible NAMD module!',
             '--gpu')
 
+    # Grab list of all available and supported modules
+    available_modules = get_available_modules()
+
     for m in module:
-        # Here we detect the mdengine (GROMACS or NAMD).
+        # Here we detect the MD engine (supported: GROMACS and NAMD).
         engine = detect_md_engine(m)
 
+        # Check the current module
+        validated_module = validate_module_name(
+            m, available_modules=available_modules)
+
+        # If we are unable to find the modules path on the system or if the
+        # user told us to skip the validation, we do so.
+        if (validated_module is None) or skip_validation:
+            console.warn('Not performing module name validation.')
```

Comment:
This warning will be printed for every module that we define. We probably should put this outside the for-loop so it is only printed once.